perf_test

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+from fio import FIO
+import md
+
+import argparse
+import itertools
+import json
+import os
+import subprocess as sp
+import tabulate
+
+# Find all unmounted NVMe devices
+def find_devices(skip_devices):
+    devs = sp.check_output(["nvme", "list", "-o", "json"])
+    devs = json.loads(devs)["Devices"]
+    devs = [d["DevicePath"] for d in devs]
+
+    devs = set(devs) - set(skip_devices)
+
+    # Remove mounted devices
+    with open("/proc/mounts") as f:
+        mounts = " ".join(f.readlines())
```

Comment:
While this probably works most of the time, things with path args or stacked filesystems could get messy, 
something like:
```
for line in f:
  dev, mount, fs, opts, dump, pass = line.split()
  #dev check ...
```
Actually checks the correct argument

Reply:
Yeah, I kind of think this is too clever by a half as well... Maybe a config file so the user can explicitly specify which devices to use or some such. Then we just need to maintain a config file on specific machines.